app/index.html

```diff
@@ -546,7 +551,7 @@ <h2 class="padded-2">Loading</h2>
 					<div class="vertical align-center">
 						<h2>Theme</h2>
 						<select data-action='change-theme'>
-							<option value="light">Light</option>
+							<option value="playful">playful</option>
```

Comment:
Theme names should be capitalized and should not replace other themes, please add the light theme option again. The ordering will probably be changed to an alphabetic one but on the mean time they should be ordered in a chronological manner, meaning new themes should go last on the list.